File: checklist/6_io/io.c
#include <stdio.h>
#include <string.h>

int main() {
    char a[100];
    int b;
    double c;
    strcpy(a, "TEXTO");
    b = 0;
    c = 0.1;
    printf("%s\n", "ESCRITA");
    printf("%s\n", a);
    printf("%d\n", b);
    printf("%lf\n", c);
    printf("%s\n", "LEITURA");
    scanf("%s", &a);
    scanf("%d", &b);
    scanf("%lf", &c);
    printf("%s\n", a);
    printf("%d\n", b);
    printf("%lf\n", c);
}

File: checklist/6_io/io.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String a;
        int b;
        double c;
        a = "TEXTO";
        b = 0;
        c = 0.1;
        System.out.println("ESCRITA");
        System.out.println(a);
        System.out.println(b);
        System.out.println(c);
        System.out.println("LEITURA");
        a = sc.nextLine();
        b = sc.nextInt();
        c = sc.nextDouble();
        System.out.println(a);
        System.out.println(b);
        System.out.println(c);
        sc.close();
    }
}

